cmd/amp/cli/cli_test.go

```diff
@@ -0,0 +1,192 @@
+package cli
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	// Amplifier
+	"github.com/appcelerator/amp/api/rpc/stack"
+	"github.com/appcelerator/amp/api/server"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type TestSpec struct {
+	fileName string
+	valid    bool
+	contents []byte
+}
+
+type commandSpec struct {
+	Cmd     string   `yaml:"cmd"`
+	Args    []string `yaml:"args"`
+	Options []string `yaml:"options"`
+}
+
+// Amplifier
+var (
+	testDir          = "./test_samples"
+	config           server.Config
+	port             string
+	etcdEndpoints    string
+	elasticsearchURL string
+	kafkaURL         string
+	influxURL        string
+	dockerURL        string
+	dockerVersion    string
+	client           stack.StackServiceClient
+	ctx              context.Context
+)
+
+const (
+	defaultPort             = ":50101"
+	etcdDefaultEndpoints    = "http://localhost:2379"
+	serverAddress           = "localhost" + defaultPort
+	elasticsearchDefaultURL = "http://localhost:9200"
+	kafkaDefaultURL         = "localhost:9092"
+	influxDefaultURL        = "http://localhost:8086"
+	dockerDefaultURL        = "unix:///var/run/docker.sock"
+	dockerDefaultVersion    = "1.24"
+)
+
+// Amplifier
```

Comment:
Amplifier set up code not necessary. There is a helper function to start the server [here](https://github.com/appcelerator/amp/blob/master/api/server/server_utils.go#L76). See `TestMain` code [here](https://github.com/appcelerator/amp/blob/master/api/rpc/service/service_test.go#L92) for an example. You've already imported the package, so you should be able to just use the helper functions like the example.